### ISSUE TITLE
Can't set button type - repurpose Button's property 'type'

### ISSUE THREAD

**Issue by richarddewit** [c0bra/svelma#21]
Currently I can't set the type of a button to `submit` or `button` or `reset`, because that property is being used for the color/style of the button. Bulma itself also calls this [Colors](https://bulma.io/documentation/elements/button/#colors) and not Type. Or do the same you've done for `outlined`/`inverted`?

### Current

```html
<!-- Svelte -->
<Button type="is-primary">Login</Button>

<!-- Rendered HTML -->
<button type="is-primary" class="button is-primary">Login</button>
```

### Expected / Wished

```html
<!-- Svelte -->
<Button type="submit" color="primary">Login</Button>
<!-- or the same as outlined/inverted -->
<Button type="submit" primary>Login</Button>

<!-- Rendered HTML -->
<button type="submit" class="button is-primary">Login</button>
```

**Comment by c0bra** [comment]
Ah, looks like buefy uses a `native-type` prop for this. I'll need to add it. Thanks for reporting.

**Comment by c0bra** [comment]
Fixed in v0.0.12
